tests/system/test_aws_s3.py

```diff
@@ -24,6 +29,7 @@
     DatastoreProfileS3,
     register_temporary_client_datastore_profile,
 )
+from mlrun.datastore.sources import ParquetSource
 from mlrun.datastore.targets import ParquetTarget
 from mlrun.utils import logger
 from tests.system.base import TestMLRunSystem
```

Comment:
please delete from _make_target_names anything that you don't need:
such as:
df_url
df_path
and so on